src/app/testing/TestFakesAdapterService.ts

```diff
@@ -3,19 +3,24 @@ import { type ReadonlyHeaders } from 'next/dist/server/web/spec-extension/adapte
 export type ThrowableAdapter<T> = {
   body: T;
   statusCode: number;
-}
+};
 
 function getTestFake(adapterName: string, fakes: Map<string, unknown> | null) {
   const fake = fakes?.get(`test-fake-${adapterName}`) as string;
 
   if (!fake) {
-    throw new Error(`Missing test fake for "${adapterName}"`);
+    throw new Error(
+      `Missing test fake for "${adapterName}" Make sure you injected a fake for this adapter in your test.`
```

Comment:
I added some text here as a reminder to check the missing injected fakes in the test - which was the root cause of the issue I shared with you in LN @niksumeiko